server/re/src/mailMS.cpp

```diff
@@ -1,10 +1,11 @@
-/// \file
+// \file
```

Comment:
Need that third forward slash for doxygen. Without it, doxygen may not produce documentation for this file.